components/builder-api/src/server/resources/projects.rs

```diff
@@ -619,66 +624,28 @@ fn toggle_privacy(req: HttpRequest,
         Err(err) => return err.into(),
     };
 
-    let project_get = format!("{}/{}", &origin, &name);
-    let project = match Project::get(&project_get, &*conn).map_err(Error::DieselError) {
-        Ok(project) => project,
+    let ops = match OriginPackageSettings::get(&GetOriginPackageSettings { origin,
```

Comment:
Might be cleaner to move toggle_privacy API into settings.rs?
Also maybe look at deprecating toggle_privacy API long term - use the update settings API instead moving forward.